$OpenBSD: patch-gophernicus_c,v 1.1 2021/02/13 14:13:37 bcallah Exp $

We don't have libwrap, and these cause issues with -fno-common

Index: gophernicus.c
--- gophernicus.c.orig
+++ gophernicus.c
@@ -30,8 +30,6 @@
  * Libwrap needs these defined
  */
 #ifdef HAVE_LIBWRAP
-int allow_severity = LOG_DEBUG;
-int deny_severity = LOG_ERR;
 #endif
 
 
